resources/sanitize.js

```diff
@@ -0,0 +1,11 @@
+// This hack allows to capture the work normally happening in a rAF. We
+// may be able to remove it if the runner improves.
+window.requestAnimationFrame = (cb) => window.setTimeout(cb, 0);
+window.cancelAnimationFrame = window.clearTimeout;
+const setTimeout = window.setTimeout;
+window.setTimeout = function (handler, timeout) {
```

Comment:
Why is this code needed?

Reply:
The idea was since we have to do it in some benchmarks we can apply it globally and not run the risk of potentially skipping work.